sched/scheduler/cluster_state.go

```diff
@@ -311,10 +313,10 @@ func (c *clusterState) update(updates []cluster.NodeUpdate) {
 			ns.timeFlaky = nilTime
 			ns.snapshotId = ""
 			if c.readyFn != nil {
-				log.Infof("Reinstating flaky node momentarily: %v (%#v), %s", ns.node.Id(), ns, c.status())
+				log.Infof("Reinstating flaky node momentarily: %v (%s), %s", ns.node.Id(), ns, c.status())
```

Comment:
can just have ns.node here, %s calls String() which calls Id()

Reply:
fixed and reverted to printing ns (all of nodeState)